client/store/modules/repository/comments/getters.js

```diff
@@ -5,3 +5,13 @@ export const getActivityComments = state => activityId => {
   const activityComments = filter(state.items, { activityId });
   return orderBy(activityComments, 'createdAt', 'desc');
 };
+
+export const getUnseenComments = (state, _, rootState) => ({ id, uid }) => {
```

Comment:
Consider destructuring within a function to make it a bit more obvious that you are passing activity (since this is comments module) or don't destructure at all (might be even more readable) 😉💅